packages/core/src/scss/themeV2/form.scss

```diff
@@ -0,0 +1,33 @@
+// =============================================================================
+// V2 Form Styles
+// Code for demo purposes only, uncomment to see a different form field style
+// =============================================================================
+
+.themeV2{
+    //.form-select,
```

Comment:
Is this intentional?

Reply:
Yes, the new theme will show up when the experiment `CHECKOUT-7962.update_font_style_on_checkout_page` is on, so making these comments just to demo how the style works.